source/configuration_manual/imap_metadata.rst

```diff
@@ -25,3 +25,53 @@ Example:
   protocol imap {
     imap_metadata = yes
   }
+
+Storing metadata in SQL dictionary
+==================================
+
+You can store metadata into a database too. This works best with dedicated table
+for storing the entries. 
+
+Database schema
+---------------
+
+.. code:: sql
+
+  CREATE TABLE metadata (
+    attr_name VARCHAR(255) NOT NULL,
+    username VARCHAR(255) NOT NULL DEFAULT '',
+    attr_value VARCHAR(65535),
+    PRIMARY KEY(attr_name, username)
```

Comment:
Shouldn't it be first username then attr_name. Otherwise there's no quick index for listing a user's all attributes.

Reply:
I can add INDEX for username separately instead.

Reply:
But why, if a single index can do it also?